mappings/net/minecraft/client/resource/ResourceDownloadThread.mapping

```diff
@@ -1,7 +1,12 @@
 CLASS net/minecraft/class_268 net/minecraft/client/resource/ResourceDownloadThread
 	FIELD field_1057 directory Ljava/io/File;
 	FIELD field_1058 client Lnet/minecraft/class_243;
+	FIELD field_1059 running Z
```

Comment:
if it has a getter and field is not public -> field should not have a prefix and getter should (1.16 tooltipcontext's `advanced`, `isAdvanced()`
otherwise prefix the field and getter (1.16 world's `isClient`, `isClient()`)

Reply:
this field is private and has no getter. no `is` required as it will only be used internally